make/autoconf/libraries.m4

```diff
@@ -136,12 +136,11 @@ AC_DEFUN_ONCE([LIB_SETUP_LIBRARIES],
     BASIC_JVM_LIBS="$BASIC_JVM_LIBS $LIBPTHREAD"
   fi
 
-  # librt for legacy clock_gettime
+  # librt
   if test "x$OPENJDK_TARGET_OS" = xlinux; then
-    # Hotspot needs to link librt to get the clock_* functions.
-    # But once our supported minimum build and runtime platform
-    # has glibc 2.17, this can be removed as the functions are
-    # in libc.
+    # Needed for the timer_* functions from librt used in JFR
```

Comment:
It isn't just JFR - CompilerThreadTimeout also uses timers. I suggest just one comment at line 139 "#librt - for timers".

Reply:
Additional to JFR I see some stuff in ASSERT coding in the compiler threads coding so it is not important for release JVMs.  But probably you are right we can simplify the comments, no need to mention old stuff and too many details.